pkg/virtctl/create/vm/vm.go

```diff
@@ -442,10 +442,10 @@ func (c *createVM) usage() string {
   # Create a manifest for a VirtualMachine with multiple volumes and inferred instancetype and preference with specified volumes
   {{ProgramName}} create vm --volume-import=type:ds,src:my-annotated-ds --volume-pvc=my-annotated-pvc --infer-instancetype=my-annotated-ds --infer-preference=my-annotated-pvc
 
-  # Create a manifest for a VirtualMachine with a specified VirtualMachineCluster{Instancetype,Preference} and cloned PVC
+  # Create a manifest for a VirtualMachine with a cloned PVC
   {{ProgramName}} create vm --volume-import=type:pvc,src:my-ns/my-pvc
 
-  # Create a manifest for a VirtualMachine with a specified VirtualMachineCluster{Instancetype,Preference} and directly used PVC
+  # Create a manifest for a VirtualMachine with a directly used PVC
```

Comment:
```suggestion
  # Create a manifest for a VirtualMachine using directly a PVC
```

Reply:
Or just "using a PVC", the "directly" here doesn't seem to add value.

Reply:
I should rephrase this. I wanted to express that it is using a PVC without cloning it.